styleguide/content/brand-identity/typography.css

```diff
@@ -143,7 +144,7 @@
 :global(.type-columns > span:nth-of-type(4)) {
   flex-basis: 25%;
   flex-grow: 1;
-  color: #716f67;
+  color: #716f67; /* stylelint-disable-line mds/colors */
```

Comment:
We use this color more than once, should we define a new color variable for it?

Reply:
A couple of things about this particular instance:

1. It is for the typography page that is going to be ripped out once we get documentation from designers
2. We are skipping our rule on this color -- we didn't add the color
3. We actually have no idea where this color is being applied.